src/SFA.DAS.AssessorService.Data.IntegrationTests/Repositories/FixtureBase.cs

```diff
@@ -0,0 +1,120 @@
+﻿using SFA.DAS.AssessorService.Data.IntegrationTests.Handlers;
+using SFA.DAS.AssessorService.Data.IntegrationTests.Models;
+using SFA.DAS.AssessorService.Domain.Consts;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SFA.DAS.AssessorService.Data.IntegrationTests.Repositories
+{
+    public class FixtureBase: IDisposable
```

Comment:
For my sanity please
```suggestion
    public class FixtureBase : IDisposable
```

Reply:
Fixed